### PR TITLE
Updated terms based on Implementing Act

### LINKED ISSUE #396: Some terms need to be aligned with EUDIW Implementing Acts
‘wallet secure cryptographic application’ means an application that manages critical assets by using the cryptographic functions provided by the wallet secure cryptographic device; 

‘wallet instance’ means the application installed and configured on a wallet user’s device or environment, which is part of a wallet unit, and that the wallet user uses to interact with the wallet unit; 

‘wallet unit’ means a unique configuration of a wallet solution that includes wallet instances, wallet secure cryptographic applications and wallet secure cryptographic devices provided by a wallet provider to an individual wallet user; 

‘wallet unit attestation’ means a data object that describes the components of the wallet unit, allow authentication and validation of those components and are cryptographically bound to wallet secure cryptographic devices;

### LINKED ISSUE #445: EUDI Wallet User must be informed about RP identity
In ARF Annex II, there are the following requirements that are not clearly explained in the specification:

**OIA_05**, After verifying and processing a PID or attestation request, the EUDI Wallet Instance SHALL inform the EUDI Wallet User about the identity of the requesting Relying Party and the requested attributes.

**RPA_06**, If Relying Party authentication succeeds, the Wallet Instance SHALL display to the User the name of the Relying Party as included in the Relying Party Instance access certificate, together with the attributes requested by the Relying Party.

### PR DESCRIPTION
**Pull request by giadas**
This PR closes #396 

We have:

- aligned the Wallet Solution related terms with the Implementing Act
- updated Status Attestation term in Status Assertion
- removed duplication
- moved the alignment with the ARF or Implementing Act in the notes.

With the new commit this PR also closes #445

### CODE REVIEW

File: docs/en/defined-terms.rst [hunk 1]
```diff
@@ -23,71 +23,106 @@ Below are the description of acronyms and definitions which are useful for furth
      - Description
      - Notes
    * - User
```
Comment (giadas) [inline docs/en/defined-terms.rst:25]: We have currently kept the "User" definition of the ARF. We need to discuss whether to use the one from the Implementing Act (‘wallet user’ means a user who is in control of the wallet unit) and whether tp keep the "Holder" term as well.

**Review by grausof — APPROVED** [review]

File: docs/en/defined-terms.rst (hunk 1 above)
Comment (peppelinux) [reply docs/en/defined-terms.rst:25]: I would keep both

**Review by peppelinux — APPROVED** [review]

File: docs/en/defined-terms.rst [hunk 2]
```diff
@@ -216,3 +249,7 @@ Acronyms
     - Personally Identifiable Information
   * - **WSCD**
     - Wallet Secure Cryptographic Device
+  * - **WSCA**
+    - Wallet Secure Cryptographic Application
+
+
```
Comment (giadas) [inline docs/en/defined-terms.rst]:
```suggestion


```

File: docs/en/defined-terms.rst [hunk 3]
```diff
@@ -23,71 +23,106 @@ Below are the description of acronyms and definitions which are useful for furth
...
-     - Differences with ARF: editorial ARF alternative terms: EUDI Wallet Provider
+     - An Organizational Entity, responsible for the management and provisioning of a Wallet Solution. 
+     - | Revised from ARF v1.4 and Implementing Act.
+       | *Differences:* editorial (use of Organizational Entity instead of public or private organisation and natural or legal person, respectively). 
+       |
+       | *Alternative terms:* EUDI Wallet Provider.
    * - Wallet Instance
-     - Instance of a Wallet Solution installed on a User’s device belonging to and which is controlled by a User. It enables the storage and management of Digital Credentials.The Wallet Instance provides graphical interfaces for User interaction with Relying Parties, PID, (Q)EAA Providers and the Wallet Provider. [Revised from ARF v1.4]
-     - Differences with ARF: editorial ARF alternative terms: EUDI Wallet Instance
+     - It is an application  installed and configured on a User's device belonging to and which is controlled by a User, as part of the Wallet Unit. The Wallet Instance provides graphical interfaces for User interaction with the Wallet Unit.
```
Comment (m-basili) [inline docs/en/defined-terms.rst]:
```suggestion
     - It is an application installed and configured on a User's device belonging to and which is controlled by a User, as part of the Wallet Unit. The Wallet Instance provides graphical interfaces for User interaction with the Wallet Unit.
```

**Review by m-basili — APPROVED** [review]

**Pull request merged**